lib/modules/begroot-widgets/views/phase-voting/ideas-list.html

```diff
@@ -141,7 +141,7 @@ <h3 class="grider-title">{{idea.title}}</h3>
 					</table>
 					{% endif %}
 					<br />
-					{% if data.widget.originalIdeaUrl %}
+					{% if data.widget.originalIdeaUrl and idea.showOriginalIdeaLink %}
```

Comment:
Or only do the check in this template: `data.widget.originalIdeaUrl and !!idea.extraData.hideOriginalIdeaLink`

Reply:
Agree! :)